### PR TITLE
Update teams-for-vdi.md

### PR DESCRIPTION
**Pull request by sadafbadar**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by sadafbadar** [comment]
When the changes will go live?

File: Teams/teams-for-vdi.md [hunk 1]
```diff
@@ -466,6 +466,7 @@ if($cleanup){
 - In Citrix environments, if the user disconnects from the Virtual Machine while Teams is running, Teams updates can result in the user to be in a non-optimized state for AV when they reconnect. We recommend that users quit Teams before they disconnect from Citrix Virtual Machine to avoid this scenario.
 - Teams should be deployed either per user or per machine. Deployment of Teams for concurrent per user and per machine is not supported. To migrate from either per machine or per user to one of these modes, follow the uninstall procedure and redeploy to either mode.
 - Azure Virtual Desktop doesn't support macOS and Linux-based clients at this time.
+- Fast tenant switch can result in calling related issues on VDI. Restarting the client will mitigate these issues.
```
Comment (JohanFreelancer9) [inline Teams/teams-for-vdi.md:469]:
```suggestion
- Fast tenant switch can result in calling-related issues on VDI. Restarting the client will mitigate these issues.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sadafbadar, kindly commit the suggested change. Many thanks.

**Comment by sadafbadar** [comment]
I committed the changes in the morning. @scanum Is there any action item for me? I cant see the changes live till now and we have an ICM waiting for the response.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by sadafbadar** [comment]
I still cant see the changes online. We have ICMs to respond. When its going to reflect on public doc?

**Comment by JohanFreelancer9** [comment]
@scanum

**Comment by sadafbadar** [comment]
@scanum Why the changes are still not reflecting on the puclic doc? We have ICMs to close and this is blocking our progress

**Pull request merged**